src/patterns/phone-numbers/index.njk

```diff
@@ -0,0 +1,119 @@
+
+When using this pattern you should:
+{{
+    onsList({
+        "itemsList": [
+            {
+                "text": 'tell users why you need a phone number and what it will be used for'
+            },
+            {
+                "text": 'make it clear what type of telephone number you need'
+            },
+            {
+                "text": 'make sure you allow the user to enter a phone number in a familiar format'
+            },
+            {
+                "text": 'help users enter a phone number in the correct format'
+            }
+        ]
+    })
+}}
+
+### Explain why you need the phone number
+Use the optional label__description with the [input component](/components/input) for hint text to reassure users why you need a phone number and what it will be used for.
+
+## Make it clear what type of phone number you need
+Use the `label__description` for hint text to let users know what type of phone number you need, for example, a UK, international or mobile phone number.
+
+## Allow all phone number formats
+The phone number field must accommodate all phone number formats. The length of the field must allow for any spaces or common characters, for example, brackets, dashes or country codes.
+
+### Validate phone numbers
+
+To help users enter a valid email address, you should:
+
+{{
+    onsList({
+        "itemsList": [
+            {
+                "text": 'check they have entered the phone number correctly'
+            },
+            {
+                "text": 'show error validation messages if they have not entered a valid phone number'
+            },
+            {
+                "text": 'allow them to paste the phone number'
+            },
+            {
+                "text": 'ask them to confirm it is correct using the <a href="/patterns/check-answers/">check answers pattern</a>'
+            }
+        ]
+    })
+}}
+
+### Error messages
+Use the [correct errors pattern](https://ons-design-system.netlify.app/patterns/error-validation) and show the error details above the phone number field if it is missing or incorrect.
+{{
+    patternlibExample({"path": "patterns/phone-numbers/examples/phone-number-error/index.njk"})
+}}
+
+#### If the phone number is missing
+Use “Enter [the required type of phone number]”, for example, “Enter a UK mobile number”.
+
+#### If a phone number is incorrect
+Use “Enter a UK mobile number in a valid format, for example, 07700 912345 or +44 7700 912345”, to give them a clear example of a valid phone number.
+Make sure any examples of phone numbers in the error message are suitable for the type of number you are asking for. {{
+    onsExternalLink({
+        "url": "https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama",
+        "linkText": "Ofcom maintains a list of numbers"
+    })
+}} reserved for use in media that you can use.
+
+### Use autocomplete 
```

Comment:
Think this should be a level 2 heading as it's not related to 'validate phone numbers'
```suggestion
## Use autocomplete 
```